packages/app-store/office365calendar/api/add.ts

```diff
@@ -1,7 +1,7 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 import { stringify } from "querystring";
 
-import { WEBAPP_URL } from "@calcom/lib/constants";
+import { WEBAPP_URL_FOR_OAUTH } from "@calcom/lib/constants";
```

Comment:
This is similar to other apps that have strict checks for redirect_URI because we use app.cal.com now for running local web app and we can only use https://localhost or a public domain(no app.cal.local), we use http://localhost:3000 for specifying the redirect URI.